Require PowerShell 7 + handle throttling errors (#785)
microsoft/finops-toolkit: docs/_automation/powershell/README.md

## 📥 Install the module

> The FinOps toolkit module requires PowerShell 7, which is supported on all major operating systems but is not installed by default. For details, see [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell). Once PowerShell has been installed, run the following:

Reviewer: @flanakin maybe we should add a note about PS Core 7 being installed by default on the Azure Cloud Shell, which should probably be the preferred prompt where to run the module.

Author: I'd be fine with that assuming we validate it can be installed first. Have you tested loading the FTK module in CloudShell? If not, we can add it later.

Author: ```suggestion
The FinOps toolkit module requires PowerShell 7, which is built into [Azure Cloud Shell](https://portal.azure.com/#cloudshell) and supported on all major operating systems. 

Azure Cloud Shell comes with PowerShell 7 and Azure PowerShell pre-installed. If you are not using Azure Cloud Shell, you will need to [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell) first and then run the following commands to install Azure PowerShell:
```

```powershell
Install-Module -Name Az.Accounts

> Install-Module -Name Az.Resources

Author: ````suggestion
Install-Module -Name Az.Resources
```

To install the FinOps toolkit module, run the following in either Azure Cloud Shell or a PowerShell client:

```powershell
````

> Import-Module -Name FinOpsToolkit

Reviewer: Should be Install-Module instead of Import-Module. Importing a module without installing it raises an error:

> Import-Module: The specified module 'FinOpsToolkit' was not loaded because no valid module file was found in any module directory.

Author: ```suggestion
Install-Module -Name FinOpsToolkit
```

> Import-Module -Name FinOpsToolkit

Author: ````suggestion
```

To install the FinOps toolkit module, run the following in either Azure Cloud Shell or a PowerShell client:

```powershell
Import-Module -Name FinOpsToolkit
````

> Import-Module -Name FinOpsToolkit

Author: ````suggestion
Install-Module -Name Az.Resources
```

To install the FinOps toolkit module, run the following in either Azure Cloud Shell or a PowerShell client:

```powershell
Import-Module -Name FinOpsToolkit
````

> Install-Module -Name FinOpsToolkit

Author: ```suggestion
Install-Module -Name FinOpsToolkit sjfda;lskdjfa ;jfd
```
